src/components/App/App.tsx

```diff
@@ -0,0 +1,33 @@
+import "./App.scss"
+import smilies from "../../assets/images/smilies.png"
+import mobilePreview from "../../assets/images/i6.png"
+import appstore from "../../assets/images/appstore.png"
+import facebook from "../../assets/images/Facebook.png"
+import twitter from "../../assets/images/Twitter.png"
+import menu from "../../assets/images/menu.png"
+
+function App() {
+  return (
+   <div className="main">
+    <div className="left-container">
+      <div className="left-container__menu">
+        <img src={menu} />
```

Comment:
img tags require an alt attribute in order to improve accesibility and as a fallback if the image doesn't load. WDYT if you add it?